pkg/provider/aws/knowledgebase/api_gateway.go

```diff
@@ -109,6 +109,97 @@ var ApiGatewayKB = knowledgebase.Build(
 			return nil
 		},
 	},
+	knowledgebase.EdgeBuilder[*resources.ApiIntegration, *resources.LoadBalancer]{
+		Expand: func(integration *resources.ApiIntegration, lb *resources.LoadBalancer, dag *core.ResourceGraph, data knowledgebase.EdgeData) error {
+			// This isnt an instance of an expanded path, rather an existing edge so ignore
+			if integration.Name != "" {
+				return nil
+			}
+			restApi, ok := data.Source.(*resources.RestApi)
```

Comment:
for my own understanding -- are the source and destination in the `EdgeData` the original, non-expanded edge's nodes?

Reply:
yup, i believe i put a godoc for that, but i will do another pass on godocs to help explain it all